### PR TITLE
[BULK EDIT - FRONTEND] refactoring of JS for summary popups

### PR DESCRIPTION
**Pull request by rusllonrails**

### CODE REVIEW

File: app/assets/javascripts/components/bulk-edit-measures-save-actions.js.coffee [hunk 1]
```diff
@@ -127,37 +107,33 @@ window.BulkEditOfMeasuresSaveActions =
...
-    $(document).on 'click', '.js-bulk-edit-of-measures-error-details-close-popup', ->
-      MicroModal.close("modal-1530613430")
-      $(".js-bulk-edit-of-measures-error-details").html("")
+  showSummaryPopup: ->
+    modal_id = "bem-save-progress-summary"
+    content = "There are no any validation errors!"
 
-      return false
+    if $(".has-validation-errors").length > 0
+      content = "Some measures are having validation errors! <br /> Please review table cells with highighted with red."
```
Comment (matthewford) [inline app/assets/javascripts/components/bulk-edit-measures-save-actions.js.coffee]: Some measures have conformance errors, please review table cells with highlighted in red

File: app/assets/javascripts/components/bulk-edit-measures-save-actions.js.coffee [hunk 2]
```diff
@@ -127,37 +107,33 @@ window.BulkEditOfMeasuresSaveActions =
...
     $(".js-bulk-edit-of-measures-submit-for-cross-check").removeClass('disabled')
     $(".js-bulk-edit-of-measures-exit").removeClass('disabled')
 
-  closeErrorDetailsPopup: ->
-    $(document).on 'click', '.js-bulk-edit-of-measures-error-details-close-popup', ->
-      MicroModal.close("modal-1530613430")
-      $(".js-bulk-edit-of-measures-error-details").html("")
+  showSummaryPopup: ->
+    modal_id = "bem-save-progress-summary"
+    content = "There are no any validation errors!"
```
Comment (matthewford) [inline app/assets/javascripts/components/bulk-edit-measures-save-actions.js.coffee]: There are no conformance errors

File: app/assets/javascripts/components/bulk-edit-measures-save-actions.js.coffee [hunk 3]
```diff
@@ -127,37 +107,33 @@ window.BulkEditOfMeasuresSaveActions =
...
-      MicroModal.close("modal-1530613431")
-      $(".js-bulk-edit-of-measures-error-details").html("")
-
-      return false
+    if window.__save_bulk_edit_of_measures_mode == "save_group_for_cross_check"
+      if $(".has-validation-errors").length > 0
+        modal_id = "bem-submit-summary-failed"
+      else
+        modal_id = "bem-submit-summary-success"
+        content = "After review you will get a feedback!"
```
Comment (matthewford) [inline app/assets/javascripts/components/bulk-edit-measures-save-actions.js.coffee]: Need to check if there is copy in the designs, but perhaps "You have submitted your work for cross-check"

Comment (rusllonrails) [reply app/assets/javascripts/components/bulk-edit-measures-save-actions.js.coffee]: @matthewford sure :+1: will update copies in next PR

**Pull request merged**